temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -112,6 +112,22 @@ service CloudService {
         };
     }
 
+    // Failover a multi-region namespace
+    rpc FailoverNamespace (FailoverNamespaceRequest) returns (FailoverNamespaceResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/namespaces/{namespace}/failover",
```

Comment:
```suggestion
            post: "/api/v1/cloud/namespaces/{namespace}/failover",
```

Sorry to come in late on this, but as part of #20 we changed this root to be `/api/v1/cloud`. Can you apply this root to the two calls here? Thanks!